### PR TITLE
feat(repo): add tf-gcp to the sdk-spec-tests action

### PR DESCRIPTION
**Pull request by tsuf239**
fixes: [#5236](https://github.com/winglang/wing/issues/5236)
see the issue for more information about the changes

## Checklist

- [ ] Title matches [Winglang's style guide](https://www.winglang.io/contributing/start-here/pull_requests#how-are-pull-request-titles-formatted)
- [ ] Description explains motivation and solution
- [ ] Tests added (always)
- [ ] Docs updated (only required for features)
- [ ] Added `pr/e2e-full` label if this feature requires end-to-end testing

*By submitting this pull request, I confirm that my contribution is made under the terms of the [Wing Cloud Contribution License](https://github.com/winglang/wing/blob/main/CONTRIBUTION_LICENSE.md)*.

### CODE REVIEW

**Comment by tsuf239** [comment]
spec tests action run:
https://github.com/winglang/wing/actions/runs/9743897058

File: examples/tests/sdk_tests/bucket/load_test.test.w [hunk 1]
```diff
@@ -2,8 +2,8 @@ bring cloud;
 
 let b = new cloud.Bucket();
 
-test "uploading many objects" {
+new std.Test(inflight () => {
   for i in 0..500 {
     b.put("test{i}", "{i}");
   }
-}
+}, timeout: 3m) as "uploading many objects";
```
Comment (hasanaburayyan) [inline examples/tests/sdk_tests/bucket/load_test.test.w:9]: I wonder if we should open an issue to provide a way to pass these props in the current test syntax

File: libs/wingsdk/src/target-tf-gcp/permissions.ts [hunk 2]
```diff
@@ -1,59 +1,42 @@
 import * as cloud from "../cloud";
 
-export function calculateBucketPermissions(ops: string[]): string[] {
+export function createBucketPermissions(ops: string[]): string[] {
```
Comment (hasanaburayyan) [inline libs/wingsdk/src/target-tf-gcp/permissions.ts:3]: Whats the reason for the name change here? We use `calculateXxxPermissions` for all other targets

**Review by hasanaburayyan — APPROVED** [review]

File: examples/tests/sdk_tests/fs/stat.test.w [hunk 3]
```diff
@@ -17,7 +20,11 @@ test "metadata()" {
     expect.equal(unixMode.test(fileStats.permissions), true);
     assert(fileStats.accessed.year >= 2023);
     assert(fileStats.modified.year >= 2023);
-    assert(fileStats.created.year >= 2023);
+   if (!isGCP) {
+    // creation time is not available on gcp
```
[the remark this answers is not in the document]
Comment (tsuf239) [reply examples/tests/sdk_tests/fs/stat.test.w:24]: True!

File: examples/tests/sdk_tests/bucket/load_test.test.w (hunk 1 above)
Comment (tsuf239) [reply examples/tests/sdk_tests/bucket/load_test.test.w:9]: we already have an old one here: https://github.com/winglang/wing/issues/2594

**Pull request merged**